main.go

```diff
@@ -311,6 +314,26 @@ func main() {
 		fail("Issue with input: %s", err)
 	}
 
+	//
+	// Fastlane session
+	fmt.Println()
+	log.Infof("Ensure cookies for Apple Developer Portal")
+
+	fs, err := session.GetSession()
+	if err != nil {
+		log.Warnf("Failed to get the session for the Apple Developer Portal, error: %s", err)
```

Comment:
It would be great to mention that this is related to the Bitrise Apple Developer Portal connection, something like:

`Failed to activate the Bitrise Apple Developer Portal connection: %s\nRead more: https://devcenter.bitrise.io/getting-started/signing-up/connecting-apple-dev-account/`

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/ba4be9a32c9c3752a21b80ac930eee7c0fd2fdb6